style.css

```diff
@@ -111,7 +127,7 @@ a {
   align-items: center;
   text-align: center;
   border-style: dashed;
-  border-color: rgb(230, 140, 140);
+  border-color: transparent;
 }
 
 .divChild2 {

```

Comment:
What do you think about placing the sound button on the same line as the play button? I think that would be more intuitive. And, if you do that, maybe make the two buttons looks more similar in style. Not sure how I feel about the round play button! lol